src/build/mod.rs

```diff
@@ -7,11 +7,13 @@ use crate::{commands, install};
 use std::path::PathBuf;
 use std::process::Command;
 
-pub fn build(target: &Target) -> Result<(), failure::Error> {
+// This is called by both build and publish. Should return a struct of whatever info we need to show success
```

Comment:
should this be a struct or a string? i'd be into a struct, but i'm not 100% certain what it would contain.

Reply:
Oops, left out a TODO. I think it will need to be a struct in the future once we have different information in the `build` output depending on project type. For now, its just going to be a string describing the result.